src/control/common/storage_types.go

```diff
@@ -38,13 +38,13 @@ func (nc NvmeControllers) String() string {
 	var buf bytes.Buffer
 
 	for _, ctrlr := range nc {
-		fmt.Fprintf(
-			&buf, "\tPCI Address:%s Serial:%s\n\tModel:%s Fwrev:%s\n",
```

Comment:
Looking at this a bit more, I also notice an inconsistency between the SCM formatting and NVMe formatting. Here, the fields are formatted with `$name:%s`, but in SCM it's `$name: %s`. It's a minor thing visually, but irritating to deal with when trying to parse output with other tools.

Reply:
I noticed that as well just after pushing, will fix as I agree it's irritating